packages/ckeditor5-utils/src/env.ts

```diff
@@ -218,14 +222,18 @@ export function isRegExpUnicodePropertySupported(): boolean {
 
 /**
  * Checks if the user agent has enabled a forced colors mode (e.g. Windows High Contrast mode).
+ *
+ * Returns `false` in environments where `window` global object is not available.
  */
 export function isMediaForcedColors(): boolean {
-	return window.matchMedia( '(forced-colors: active)' ).matches;
+	return globalVar.window.matchMedia ? globalVar.window.matchMedia( '(forced-colors: active)' ).matches : false;
 }
 
 /**
  * Checks if user enabled "prefers reduced motion" setting in browser.
+ *
+ * Returns `false` in environments where `window` global object is not available.
  */
 export function isMotionReduced(): boolean {
-	return window.matchMedia( '(prefers-reduced-motion)' ).matches;
+	return globalVar.window.matchMedia ? globalVar.window.matchMedia( '(prefers-reduced-motion)' ).matches : false;
```

Comment:
Because it was a getter, this function didn't crash in described scenario because the getter was never called. However, the getter would still crash if called in an environment without `window`. To make the solution a bit more future-proof, I decided to add a check for `globalVar.window`.

Reply:
I believe we should mock all methods and fields in `globalVar.window` that we utilize in our codebase. Centralizing them would eliminate the need to repeat this logic in multiple places. However, perhaps this task could be deferred until later.